windowsdoc.md

```diff
@@ -0,0 +1,67 @@
+## **Windows Compatible Cwltool**  
+Windows Compatible cwltool means that now you can create and run your workflows usinng cwltool on windows under a docker container.
```

Comment:
- Mention which versions of Windows are supported (at least Windows 10 professional)
- Add a link to Docker installation instructions for Windows (https://docs.docker.com/docker-for-windows/)